src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1351,6 +1351,24 @@ public function findTaggedServiceIds(string $name, bool $throwOnAbstract = false
         return $tags;
     }
 
+    /**
+     * Find definitions of value-objects by tag.
+     *
+     * @return array<string, array> An array of tags with the tagged classes as key, holding a list of attribute arrays
+     */
+    public function findTaggedValueObject(string $name): array
```

Comment:
I'd still do the $autoExclude (adding the exclude tag only if it's not there already)